lib/acx/enforcer_server.ex

```diff
@@ -340,12 +341,32 @@ defmodule Acx.EnforcerServer do
   # Returns a tuple used to register and lookup an enforcer process
   # by name
   defp via_tuple(ename) do
-    {:via, Registry, {Acx.EnforcerRegistry, ename}}
+    Application.get_env(:acx, :registry)
+    |> case do
+      Registry ->
+        {:via, Registry, {Acx.EnforcerRegistry, ename}}
+
+      Horde.Registry ->
+        {:via, Horde.Registry, {Acx.Horde.HordeRegistry, ename}}
```

Comment:
I think Acx.EnforcerRegistry is just a name. I searched the library and I don't see it as a module or anything so I think it can be anything, which makes me wonder if you could simply do `{:via, Application.get_env(:acx, :registry, Registry), {Acx.EnforcerRegistry, ename}}`